curriculum/challenges/english/blocks/workshop-registration-form/60fab8367d35de04e5cb7929.md

```diff
@@ -7,48 +7,44 @@ dashedName: step-27
 
 # --description--
 
-Add the text `I accept the terms and conditions` immediately after the `input` element in the newly added `label`. Then link the text `terms and conditions` to the following location:
+Add an anchor `a` with the text `Read our terms and conditions` before the newly added `label`. Link the text `terms and conditions` to:
 
 ```md
 https://www.freecodecamp.org/news/terms-of-service/
 ```
 
-# --hints--
-
-You should add `I accept the terms and conditions` text to the `label` following the third `fieldset`.
+Then add the text `I accept the terms and conditions` immediately after the input element in the label.
 
-```js
-assert.equal(document.querySelector('fieldset:nth-child(3) + label')?.innerText.trim(), 'I accept the terms and conditions');
-```
+# --hints--
 
-You should use an `a` element to link to the terms and conditions.
+You should use an `a` element to link to the terms and conditions with the text `Read our terms and conditions` before the `label` that follows the third `fieldset`.
 
 ```js
-assert.exists(document.querySelector('fieldset:nth-child(3) + label a'));
+assert.exists(document.querySelector('fieldset:nth-child(3) + a')?.innerText.trim(), 'Read our terms and conditions');
```

Comment:
```suggestion
assert.match(document.querySelector('fieldset:nth-child(3) + a')?.innerText.trim().replaceAll(/\s+/g, ' '), /Read our terms and conditions\.?/i);
```

let's make things easier for everyone, and make the dot at the end optional, and don't care about multiple spaces.